Template/NLO/Cards/run_card.dat

```diff
@@ -66,6 +66,18 @@
  %(ebeam1)s   = ebeam1  ! beam 1 energy in GeV
  %(ebeam2)s   = ebeam2  ! beam 2 energy in GeV
 #***********************************************************************
+# Choice of the collision system:
+#    True/False = asymmetric / symmetric
+#  
```

Comment:
Should not you rather comment on what it does  (and how)?
Otherwise people will be confuse

Reply:
I agree. I will add more details about this “asymmetry” case.